### PR TITLE
Feature/monitoring ceiling

### PR DESCRIPTION
**Pull request by claireriley**
@ArjunSingh-PM — couple additions for the monitoring 1.3 release. Let me know what you think about formatting/flow. I felt like it made the most sense to include the bulk of it under "Usage limits" with our current setup, however I added the note formatting as an option to differentiate it from the typical usage limits folks would encounter.

I was unclear on the final decision for notification format/recipient as the limits in the product spec seemed to vary a bit from the design spec, please let me know how that should be updated (or if you think that line should not be included).

### CODE REVIEW

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 1]
```diff
@@ -85,7 +85,19 @@ You can use Postman monitoring with any paid or free plan type.
 
 ### Usage limits
 
-You can find your account type's usage limits by referring to your [usage overview](https://go.pstmn.io/postman-account-limits).
+You can find your account type's monthly usage limits by referring to your [usage overview](https://go.pstmn.io/postman-account-limits).
+
+> Postman maintains ceiling limits on various team and user actions to ensure the overall performance and availability of monitoring.
```
Comment (godfrzero) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Should we also mention that these are the _default_ limits?

File: src/pages/docs/postman/monitors/setting-up-monitor.md [hunk 2]
```diff
@@ -107,6 +107,8 @@ You will need to give your new monitor a name and designate the collection you w
 
 [![create monitor](https://assets.postman.com/postman-docs/setting-up-a-monitor.jpg)](https://assets.postman.com/postman-docs/setting-up-a-monitor.jpg)
 
+> Postman maintains ceiling limits on various team and user actions, including monitor creation. For more information, see [Usage limits](/docs/postman/monitors/intro-monitors/#usage-limits).
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/postman/monitors/setting-up-monitor.md]: It's relative it'll work in prod!

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 3]
```diff
@@ -85,7 +85,19 @@ You can use Postman monitoring with any paid or free plan type.
...
+
+> Postman maintains default limits on various team and user actions to ensure the overall performance and availability of monitoring.
+>
+> These include the following limits:
+>
+> * Maximum number of active and paused monitors per team = 300
+> * Maximum parallel runs of multiple monitors = 500
+> * Maximum parallel runs of a single monitor = 200
+>
+> Postman will notify team admins via email if your team encounters these limits, as well as notify users in the app or web dashboard UI.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: I'd just change "notify" to "notifying" here

Comment (SueSmith) [reply src/pages/docs/postman/monitors/intro-monitors.md]: would "team members" work as a replacement for "users" here?

maybe lose "UI" at the end too?

**Pull request merged**